styles.css

```diff
@@ -97,22 +142,26 @@ ul {
 
 .work-card {
   width: 100%;
-  border: 1px solid #dfe1e6;
+
+  /* border: 1px solid #dfe1e6; */
```

Comment:
Great work @fullstop125 🤝

- [ ] [Optional] Thanks for implementing all the changes requested. You may remove commented code from your stylesheet. It is not regarded as a good practice to have commented code in your project.

Reply:
thank you for the honest feedback @1GORDON I appreciate